src/axes/axis.ts

```diff
@@ -104,6 +104,33 @@ export class Axis<D> extends Component {
     this._scale.offUpdate(this._rescaleCallback);
   }
 
+  /**
+   * Gets the tick label data at a particular point.
+   *
+   * @param {Point} point
+   */
+  public tickLabelAt(point: Point): D {
+    // only measure in the direction of the axis.
+    const pointValue = this.isHorizontal() ? point.x : point.y;
+    const tickValues = this._getTickValues();
+
+    const axisScale = this._scale;
+    let tickLabel: D;
+    this._tickLabelContainer.selectAll<SVGGElement, any>("." + Axis.TICK_LABEL_CLASS)
+      .each(function (data: D, index: number) {
```

Comment:
nit: we can do binary search to find it in log n; although if we can avoid doing `getBBox()` at all we can probably not care about the difference